editor/plugins/animation_blend_tree_editor_plugin.cpp

```diff
@@ -456,16 +456,22 @@ void AnimationNodeBlendTreeEditor::_delete_request(const String &p_which) {
 	undo_redo->commit_action();
 }
 
-void AnimationNodeBlendTreeEditor::_delete_nodes_request() {
+void AnimationNodeBlendTreeEditor::_delete_nodes_request(const TypedArray<StringName> &p_nodes) {
```

Comment:
In 3.x there's no typed arrays.
```suggestion
void AnimationNodeBlendTreeEditor::_delete_nodes_request(const Array &p_nodes) {
```
also you need to add the same parameter to the declaration of this method:
https://github.com/godotengine/godot/blob/01574a566ff1f1a7270bd986b226c1bd351f894e/editor/plugins/animation_blend_tree_editor_plugin.h#L106

Just check what was done in #61145. Also... in theory before pushing changes to the PR you should compile and test them on your end to ensure it actually works. :wink:

Reply:
And to apply a suggestion like this you can do it like before (click button like "commit suggestion" on GitHub, end up with 2 commits, then fixup+ force push), or just manually ammend the changes to your local commit and force push.

Reply:
Sounds good! My bad on the compiling. I just amend the changes. Tested the build, and was able to compile and verified the bug was resolved.

Reply:
Great! So just a matter of passing the tests and it should be good to merge. :slightly_smiling_face: